pom.xml

```diff
@@ -0,0 +1,22 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>legacy.postgresql</groupId>
+    <artifactId>postgresql</artifactId>
+    <version>9.1-901.jdbc4</version>
+
+    <properties>
+        <maven.compiler.source>1.6</maven.compiler.source>
```

Comment:
Converted to use Maven instead of ANT. Set the source level to 1.6 and runtime level to 1.7 (Maybe this can be 1.8?)